mqtt/mqtt-edgehub/src/tls.rs

```diff
@@ -1,16 +1,26 @@
 use chrono::{DateTime, NaiveDateTime, ParseResult, Utc};
+use native_tls::Identity;
 use openssl::{asn1::Asn1TimeRef, pkcs12::Pkcs12, pkey::PKey, stack::Stack, x509::X509};
+use std::{
+    convert::TryFrom,
+    fs,
+    path::{Path, PathBuf},
+};
 
 /// Identity certificate that holds server certificate, along with its corresponding private key
 /// and chain of certificates to a trusted root.
+#[derive(Debug)]
 pub struct ServerCertificate {
     der: Vec<u8>,
     not_before: DateTime<Utc>,
     not_after: DateTime<Utc>,
 }
 
 impl ServerCertificate {
-    pub fn try_from<S, K>(certificate: S, private_key: K) -> Result<Self, ServerCertificateError>
+    pub fn from_pem_pair<S, K>(
```

Comment:
Will this method be used anywhere except tests?

Reply:
no, it will be used when server cert downloaded from edgelet.